10up/block-components
# fix(icon picker): fix for missing icons in the editor icon picker

### Description of the Change

Fixes missing icons in Safari.



Closes #274

### How to test the Change

View the Icon Picker component in Safari using the editor. Verify the icons appear.

### Changelog Entry

> Fixed - Missing icons in the IconPicker on Safari

### Credits

Props @bmarshall511 


### Checklist:
